roadie-rails.gemspec

```diff
@@ -22,11 +22,11 @@ Gem::Specification.new do |spec|
   spec.extra_rdoc_files = %w[README.md Changelog.md LICENSE.txt]
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "railties", ">= 5.1", "< 7.1"
+  spec.add_dependency "railties", ">= 5.1"
```

Comment:
```suggestion
  spec.add_dependency "railties", ">= 5.1", "< 8.0"
```

Reply:
I disagree.  This just opens up the same timebomb for next time.  I think maximum versions should only be set once it's _known_ that they don't work.  Until proven otherwise, we shouldn't block it from being installed since it's just as likely to keep working on 8.0.